<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="orbitView">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/WorldView.png</normaloff>icons/WorldView.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="planView">
         <property name="styleSheet">
          <string notr="true">QPushButton{
margin:0px;
border:0px;
padding:0px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/PlanView.png</normaloff>icons/PlanView.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Z Exagerration</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QDoubleSpinBox" name="zfak">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="nsBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>756</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Nodestrings</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="1">
          <widget class="QComboBox" name="nodestrings"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="nsFilter"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QTableWidget" name="nsTable">
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Code</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nodestrings</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="nsZoom">
           <property name="text">
            <string>zoomen auf</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>756</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 2</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>900</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gfxCanvas"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
